src/components/form-editor/form-editor.component.tsx

```diff
@@ -392,11 +464,12 @@ const FormEditorContent: React.FC<TranslationFnProps> = ({ t }) => {
             <TabList aria-label="Form previews">
               <Tab>{t('preview', 'Preview')}</Tab>
               <Tab>{t('interactiveBuilder', 'Interactive Builder')}</Tab>
+              <Tab>{t('translationBuilder', 'Translation Builder')}</Tab>
               {form && <Tab>{t('auditDetails', 'Audit Details')}</Tab>}
             </TabList>
             <TabPanels>
               <TabPanel>
-                <FormRenderer schema={schema} isLoading={isLoadingFormOrSchema} />
+                <FormRenderer schema={mergedSchemaForPreview || schema} isLoading={isLoadingFormOrSchema} />
```

Comment:
we don't need the `|| schema` fallback here.  

The mergedSchemaForPreview will always be either,

1. The translated version of the schema or
2. The original schema